numpy/core/tests/test_regression.py

```diff
@@ -323,6 +323,13 @@ def bfb():
         assert_raises(ValueError, bfa)
         assert_raises(ValueError, bfb)
 
+    def test_fill_readonly(self):
+        # Ticket #22922
+        with pytest.raises(ValueError):
+            a = np.zeros(11)
+            a.setflags(write=False)
```

Comment:
Please move this out of the context, best to make the test as specific as possible?  Also we usually put `gh-22922` (and maybe "issue" rather than ticket, but it doesn't matter).  "Ticket #..." sounds like it predates github :). 

I would slightly prefer to move the test into `test_multiarray.py` which has a few `fill` related tests.